Sources/HummingbirdFoundation/Files/FileMiddleware.swift

```diff
@@ -44,11 +44,11 @@ public struct HBFileMiddleware: HBMiddleware {
         _ rootFolder: String = "public",
         cacheControl: HBCacheControl = .init([]),
         searchForIndexHtml: Bool = false,
-        application: HBApplication
+        application: HBApplicationBuilder
```

Comment:
I don't see why we need to pass in the `HBApplication` or `HBApplicationBuilder` here if we only care about the thread pool. It can be a `convenience init` of course, but I think this is a bit excessive.

Reply:
fixed